### ISSUE TITLE
Comment blocks used to add a '*' at the beginning of each new line when "Enter"

### ISSUE THREAD

**Issue by Pilchie** [dotnet/roslyn#2077]

**Comment by JimF42** [comment]
Let me give a tad more detail (as outlined in my [connect ticket](https://connect.microsoft.com/VisualStudio/feedback/details/1573358/c-multiline-comments)--before I knew about this opened issue).

> In versions of VS prior to 2015 multiline comments (/* … */) in c# automatically added an asterisk and space as new lines were added. Now, when starting a multiline comment and pressing enter no asterisk is added.
> 
> Please restore this functionality or at least make it an option.
> Jim

**Comment by diryboy** [comment]
I found that the file `AbstractDocumentationCommentCommandHandler.cs` within `src\EditorFeatures\Core\Implementation\DocumentationComments\` can serve as an example of implementing this.

However it's too bad the EditorFeatures cannot be deployed for open source debugging, which makes this item not really up for grabs, at least this time.

I saw discussions about this in #981 #3998 #1041 so it's glad to know Microsoft is solving this problem.
